docker/worker/worker_test.py

```diff
@@ -145,15 +145,6 @@ def setUp(self):
         public=False)
     allocated_bug.put()
 
-    should_be_deleted = osv.AffectedCommit(
-        id='OSV-2020-1337-abcd',
-        bug_id='OSV-2020-1337',
-        commit='abcd',
-        project='project',
-        ecosystem='ecosystem',
-        public=False)
-    should_be_deleted.put()
-
```

Comment:
Is there no equivalent new version of this?
(I don't really know what this was testing...)

Reply:
Good catch, this was just checking if the new analysis overwrites previous results. I'll add the new equivalent here and a comment to explain.